### ISSUE TITLE
(experimental) conda package for Windows

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/mdanalysis#2116]
Even though we do not fully support Windows (yet) we would like to have (experimental) conda packages for Windows.

- Code that is not working yet should be disabled under Windows (example: GSD Reader).
- Have MDA issue an initial warning that Windows support is experimental.

(This came out of a discussion regarding the Workshop. We would like to test if the basic examples work under Windows.)

**Comment by orbeckst** [comment]
@kain88-de @tylerjereddy @richardjgowers do you know how to build Windows packages on conda-forge?

**Comment by tylerjereddy** [comment]
I'll make a note about this.. I think I'm slotted in as release manager for the next SciPy release so should probably brush up anyway.

**Comment by tylerjereddy** [comment]
I'll make a note about this -- I think I'm slotted in to do the next major SciPy release in November so should likely brush up anyway

**Comment by richardjgowers** [comment]
The optimist in me hopes we can just delete the skip windows line here:

https://github.com/conda-forge/mdanalysis-feedstock/blob/master/recipe/meta.yaml#L16

Then maybe add any fixes that @tylerjereddy has had to use on our windows CI?

**Comment by kain88-de** [comment]
We remove the windows limitation in the build. There should be a line with
skip. Afterwards you need to rerender the ci scripts with conda-smithy.

**Comment by richardjgowers** [comment]
So hopefully the build passes without too much drama.  I'm thinking I should go through and "deactivate" the modules that don't work for Windows (like we recently did for GSD).  I think this will be most of the ascii based multi frame readers(!).

But then everything accessible within the module should work 100%

**Comment by kain88-de** [comment]
oh yeah but I don't know how to have optional requirements in a conda package. We have to make those truly optional with correct error messages in MDAnalysis. So I assume now the build will fail until the dependencies are removed.

**Comment by richardjgowers** [comment]
@kain88-de we've got the code in place to handle optional GSD, so it's just the recipe, I've tried something...

**Comment by orbeckst** [comment]
🍺

**Comment by orbeckst** [comment]
Do we only support Python 3 on Windows?

https://anaconda.org/conda-forge/mdanalysis/files

**Comment by richardjgowers** [comment]
@orbeckst yep: https://github.com/MDAnalysis/mdanalysis/pull/2093#discussion_r227438117
